src/nvim/CMakeLists.txt

```diff
@@ -97,10 +97,10 @@ file(MAKE_DIRECTORY ${GENERATED_INCLUDES_DIR})
 file(MAKE_DIRECTORY ${LINT_SUPPRESSES_ROOT})
 file(MAKE_DIRECTORY ${LINT_SUPPRESSES_ROOT}/src)
 
-file(GLOB NVIM_SOURCES *.c)
-file(GLOB NVIM_HEADERS *.h)
-file(GLOB EXTERNAL_SOURCES ../xdiff/*.c ../mpack/*.c ../cjson/*.c)
-file(GLOB EXTERNAL_HEADERS ../xdiff/*.h ../mpack/*.h ../cjson/*.h)
+glob_wrapper(NVIM_SOURCES *.c)
+glob_wrapper(NVIM_HEADERS *.h)
```

Comment:
That and because not all `*.c` files should be in `NVIM_SOURCES` and not all `*.h` files should be in `NVIM_HEADERS`. You'd need to exclude files afterwards so feels like complexity-wise is same.